.travis.yml

```diff
@@ -5,12 +5,12 @@ branches:
   - master
 before_install: bash etc/ci-setup.sh
 install: true
-script: eval $TEST_CMD
+script: /tmp/testrunner/bin/run -c . && sh etc/hatchet.sh
 after_script: if [ -n "$HEROKU_API_KEY" ]; then heroku keys:remove $USER@`hostname`; fi
 env:
   global:
   - IS_RUNNING_ON_TRAVIS=true
   - SHUNIT_HOME="/tmp/shunit2-2.1.6"
   matrix:
-  - TEST_CMD='sh etc/hatchet.sh'
-  - TEST_CMD="/tmp/testrunner/bin/run -c ."
+  - HEROKU_TEST_STACK='heroku-16'
```

Comment:
i'm worried about hatchet clobbering itself across these runs (i often push to clojure, scala, java, jvm at the same-ish time). I'm going to test things out with just the two, and then add cedar-14 if everything looks ok.

Reply:
Sounds reasonable to me 👍 

In case it's relevant: with Node I occasionally see failures on one of the stacks, and you can re-run just that one with a click.